### PR TITLE
Issue #391: default webcam fps

### PR DESCRIPTION
**Pull request by dennisvang**
- add condition to ensure default fps for webcam is *only* applied on macOS
- add note about supported webcam framerates in ffmpeg docstring

### CODE REVIEW

File: imageio/plugins/ffmpeg.py [hunk 1]
```diff
@@ -293,8 +295,8 @@ def _open(
             regex_match = re.match(r"<video(\d+)>", self.request.filename)
             if regex_match:
                 self.request._video = self.request.filename
-            # Specify input framerate?
-            if self.request._video:
+            # Specify input framerate? (only on macOS)
+            if self.request._video and platform.system().lower() == "darwin":
                 if "-framerate" not in str(self._arg_input_params):
                     self._arg_input_params.extend(["-framerate", str(float(fps or 30))])
```
Comment (almarklein) [inline imageio/plugins/ffmpeg.py:301]: Let's set the framerate to 15 in this case, which seems better supported on MacOS. See  #525 and the snippet you posted.

**Comment by almarklein** [comment]
This looks good. I wonder whether in the mean time, not specifying the framerate works on MacOS too. I cannot test this myself right now thought.

**Comment by almarklein** [comment]
This PR relates to #391, #510 and #525

File: imageio/plugins/ffmpeg.py (hunk 1 above)
Comment (dennisvang) [reply imageio/plugins/ffmpeg.py:301]: Sounds good. I'll get right on it.

**Comment by dennisvang** [comment]
> This looks good. I wonder whether in the mean time, not specifying the framerate works on MacOS too. I cannot test this myself right now thought.

Can't test this myself either. Don't have any apples.

**Review by almarklein — APPROVED** [review]

**Comment by FirefoxMetzger** [comment]
My girlfriend has a Mac ... I will see if I can give this a shot during the weekend.

@almarklein I just realized that our PR templates don't work, so we don't get reminders for prefixing/tagging squash-commit messages with `FEAT:` (new feature) or `BUG:` (bugfix) so that the CD can pick it up and know what to do with the commit. There are other tags, too, which would be neatly documented in the PR template:
https://github.com/imageio/imageio/blob/master/.github/PULL_REQUEST_TEMPLATE/default.md

buuut ... that doesn't seem to be working right now. I'll add investigating that to the TODO list.

**Pull request merged**